src/apiCalls.js

```diff
@@ -0,0 +1,10 @@
+
+
+function fetchData(data) {
+  return fetch(`http://localhost:3001/api/v1/${data}`)
```

Comment:
I appreciate that you made this function dynamic/re-useable by interpolating the data. Nice work!